### PR TITLE
chore: add coverage

### PR DESCRIPTION
**Pull request by hacksparrow**
Add coverage to the project.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,9 +8,10 @@
   "author": "IBM Corp.",
   "main": "index.js",
   "scripts": {
+    "coverage": "nyc report --reporter=text-lcov | coveralls",
```
Comment (bajtos) [inline package.json]: I think you need to add `coveralls` to `devDependencies` too.

Comment (bajtos) [reply package.json]: See https://travis-ci.org/strongloop/loopback-connector-mongodb/jobs/547095364#L3364

```
$ npm run coverage
> loopback-connector-mongodb@4.2.0 coverage /home/travis/build/strongloop/loopback-connector-mongodb
> nyc report --reporter=text-lcov | coveralls
sh: 1: coveralls: not found
events.js:183
      throw er; // Unhandled 'error' event
      ^
Error: write EPIPE
    at WriteWrap.afterWrite [as oncomplete] (net.js:868:14)
```

Comment (hacksparrow) [reply package.json]: Thanks

**Comment by hacksparrow** [comment]
@bajtos fixed it, thanks.

Anyone else?

**Comment by bajtos** [comment]
@hacksparrow when are we going to see Coveralls in the checks?

Screen Shot 2019-06-18 at 16 46 16

I see very little value in this pull request if Coveralls does not report coverage change for every pull request.

**Comment by emonddr** [comment]
> I see very little value in this pull request if Coveralls does not report coverage change for every pull request. 

@hacksparrow  

What is expected to show up in the picture @bajtos  provided?
The screenshot doesn't show all the tiny builds in the picture he provided. Could it be out of frame?

**Comment by b-admike** [comment]
@slnode test please

**Comment by b-admike** [comment]
@hacksparrow @bajtos I've enabled coveralls check for merging to `master`. Should we do the same for other branches like `3.x` or `2.x`?

**Comment by b-admike** [comment]
> > I see very little value in this pull request if Coveralls does not report coverage change for every pull request.
> 

> @hacksparrow
> 
> What is expected to show up in the picture @bajtos provided?
> The screenshot doesn't show all the tiny builds in the picture he provided. Could it be out of frame?

This is what we want to see:
Screen Shot 2019-06-19 at 10 56 19 AM

**Comment by hacksparrow** [comment]
Thanks @b-admike

**Comment by bajtos** [comment]
> I've enabled coveralls check for merging to master. Should we do the same for other branches like 3.x or 2.x?

Yes please 👍

**Pull request merged**